spec/spec_helper.rb

```diff
@@ -0,0 +1,23 @@
+require 'bundler/setup'
+Bundler.setup
+
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path("../dummy/config/environment.rb",  __FILE__)
+require 'rspec/its'
+require 'sidekiq'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
+require 'wp-connector'
+
+Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
+
+RSpec.configure do |config|
+  config.filter_run focus: true
+  config.run_all_when_everything_filtered = true
+  config.order = 'random'
+
+  config.before(:each) do | example |
```

Comment:
Unused block argument - `example`. You can omit the argument if you don't care about it.<br>Space before first block parameter detected.<br>Space after last block parameter detected.